airbyte-config/models/src/main/java/io/airbyte/config/Configs.java

```diff
@@ -72,32 +72,44 @@
   String getWebappUrl();
 
   // Jobs
-  int getMaxSyncJobAttempts();
+  int getSyncJobMaxAttempts();
```

Comment:
@jrhizor what do you think about these proposed names? The ones that haven't been modified are in comments above the current name.

Generally did away with the `worker` terminology as I feel it's confusing since we have an existing worker deployment and these are separate.

We do add `worker` as part of the job pod's name so I'm thinking we can drop that to make things clearer.

Reply:
Looks good to me.